app/views/posts/show.html.erb

```diff
@@ -1 +1,26 @@
-<h1>Post with the user id</h1>
\ No newline at end of file
+<% @posts.each do |post| %>
+<ul class="main-container">
+ <li class="blog">
+      <a href="<%= user_post_path(post.author_id, post.id) %>" class="blog">
+        <h2><%= post.title %> by <%= User.find(post.author_id).name %></h2>
```

Comment:
- [ ] [Optional] Same here, It would be better professionally if you used the `link_to` method instead of the regular anchor tag `<a>`.